src/Azure.Functions.Cli/Helpers/DotnetHelpers.cs

```diff
@@ -155,33 +174,85 @@ public static string GetCsprojOrFsproj()
             }
         }
 
-        private static async Task TemplateOperation(Func<Task> action)
+        private static Task TemplateOperation(Func<Task> action, WorkerRuntime workerRuntime)
         {
             EnsureDotnet();
+
+            if (workerRuntime == WorkerRuntime.dotnetIsolated)
+            {
+                return IsolatedTemplateOperation(action);
+            }
+            else
+            {
+                return WebJobsTemplateOpetation(action);
+            }
+        }
+
+        private static async Task IsolatedTemplateOperation(Func<Task> action)
+        {
+            try
+            {
+                await UninstallWebJobsTemplates();
```

Comment:
Why is this needed? Is there an identity conflict?

Reply:
As discussed offline, we will follow-up with a separate PR as changing the ids require confirmation with tooling partners and we will look into that.